packages/backend-common/src/scm/git.ts

```diff
@@ -150,12 +150,19 @@ export class Git {
     });
   }
 
-  async init({ dir }: { dir: string }): Promise<void> {
+  async init({
+    dir,
+    defaultBranch = 'master',
+  }: {
+    dir: string;
+    defaultBranch: string;
```

Comment:
I /think/ that this should be `defaultBranch?: string` so then typescript knows that you don't have to specify it. The API report still seems to suggest that it's required when it should not be :)

Reply:
Thanks @benjdlambert , just updated the code.

Reply:
Where would this be passed, from the UI or can this be set in the app configuration?

Reply:
Hey @cal5barton, you can pass in the default branch to the publish:github action. so you will be able to do something like this in your template to change the default branch:

```
- id: publish
      name: Publish
      action: publish:github
      input:
        allowedHosts: ['github.com']
        description: 'This is {{ parameters.name }}'
        repoUrl: '{{ parameters.repoUrl }}'
        defaultBranch: 'main' << Add this to change your default branch.
```